server/export.py

```diff
@@ -50,7 +50,8 @@ def export_predictions(solution_task):
     # copy predictions from original scoring output path to the expected D3M location
     solution_results_dir = pathlib.Path(D3MOUTPUTDIR, 'predictions', str(solution_task.id))
     solution_results_dir.mkdir(parents=True, exist_ok=True)
-    original_preds_path = utils.make_preds_filename(solution_task.id)
+    # TODO solution_task.output_keys is none, but make_preds_filename requires it. 
+    original_preds_path = utils.make_preds_filename(solution_task.id, solution_task.output_keys)
```

Comment:
solution_task.output_keys on search is none (should also only be one key per file?). Where is that supposed to be filled in?

Reply:
The export is a bit of a weird beast. Im surprised it needs output keys since it outputs a pipeline that should be fully specified (ie the result of a search solutions).

Reply:
it looks like it started requiring it when multiple output keys started being supported. https://github.com/uncharted-distil/distil-auto-ml/commit/f023e13c9357fad809539a7dc955377bca936b45#diff-04d2c9518498da64bfa9db44c6211645R62

Reply:
@BalazsHoranyi Update the `utils.make_preds_filename` function to have the `output_keys` argument default to `None`, and then within the function, generate the filename with just the solution task ID if it is unset.